.github/CODEOWNERS

```diff
@@ -14,7 +14,7 @@
 # Service teams
 /sdk/appconfiguration/ @annelo-msft
 
```

Comment:
Please include the generation metadata file update (output from generate.ps1) here: `eng\mgmt\mgmtmetadata\<rpName>.txt`

Reply:
@markcowl @AlexanderSher i added the meta data file into the project.